test/distributions/test_distributions.py

```diff
@@ -5062,7 +5074,10 @@ def f(*values):
             xfail = [
                 Cauchy,  # aten::cauchy(Double(2,1), float, float, Generator)
                 HalfCauchy,  # aten::cauchy(Double(2, 1), float, float, Generator)
-                VonMises  # Variance is not Euclidean
+                VonMises,  # Variance is not Euclidean
+                Exponential,  # mkl implementation path on intel cpu will produce diffrent results from jit.trace
```

Comment:
What's the reason MKL implementation produces different result from jit.trace? It looks a regression to me with this PR change.

Reply:
Jit is supported for ._exponential() and I removed the JIT WORKAROUND for exponential in  [   def rsample(self, sample_shape=torch.Size()):](https://github.com/pytorch/pytorch/blob/67fbd940bae60e4392fb72eb495d51f6e0261260/torch/distributions/exponential.py#L58) .  Now MKL implementation will produce same results.